src/store/reducers.js

```diff
@@ -26,6 +27,79 @@ function computeRebootStats(remediation) {
   };
 }
 
+export const remediationSystems = ({ LOAD_ENTITIES_FULFILLED }) =>
+  applyReducerHash({
+    [ACTION_TYPES.SELECT_ENTITY]: (state, { payload }) => {
+      const selected = state.selected || new Map();
+      if (payload.selected) {
+        if (payload.id === 0) {
+          state?.rows?.forEach((row) => selected.set(row?.id, row));
+        } else {
+          const selectedRow = state?.rows?.find(
+            ({ id } = {}) => id === payload.id
+          );
+          selected.set(payload.id, { ...(selectedRow || {}), id: payload.id });
+        }
+      } else {
+        if (payload.id === 0) {
+          state.rows.forEach((row) => selected.delete(row.id));
+        } else if (payload.id === -1) {
+          selected.clear();
+        } else {
+          selected.delete(payload.id);
+        }
+      }
+
+      return {
+        ...state,
+        selected: new Map(selected),
+      };
+    },
+    [`${LOAD_ENTITIES_FULFILLED}`]: (state) => {
+      return {
+        ...state,
+        rows: state.rows.map(({ id, ...row }) => ({
+          id,
+          ...row,
+          selected: !!state.selected?.get(id),
+        })),
+        columns: [
+          ...state.columns.filter(({ key }) =>
+            ['display_name', 'tags'].includes(key)
+          ),
+          {
+            key: 'issues',
+            title: 'Issues',
+            // eslint-disable-next-line react/display-name
+            renderFunc: (issues, id, { display_name }) => (
+              <IssuesColumn
+                issues={issues}
+                id={id}
+                displayName={display_name}
+              />
+            ),
+            props: { width: 15 },
+          },
+          {
+            key: 'rebootRequired',
+            title: 'Reboot required',
+            // eslint-disable-next-line react/display-name
+            renderFunc: (rebootRequired) => (
```

Comment:
Can we not store react components in redux?

Reply:
I'll try looking at `cellTransformer` so we don't have to use the `renderFunc`

Reply:
Mhh `cellFormatters` wouldn't help us either as that's the same as `renderFunc` and inventory expects columns to be defined in redux and not trough props.